docs/pages/repo/docs/handbook/dev.mdx

```diff
@@ -95,13 +95,22 @@ This means that users of your `dev` task _don't need to worry about codegen or m
 
 ## Running `dev` only in certain workspaces
 
-To run a `dev` task in only certain workspaces, you should use the [`--filter` syntax](/repo/docs/core-concepts/monorepos/filtering). For example:
+Let's assume you want to run the `dev` task in the `docs` workspace, located at `<root>/apps/docs`.
+With a global install of `turbo`, e.g. via `npm install --global turbo`, you can do
```

Comment:
Should we use the `selected-pkg-manager` tab here to provide all of the possible install commands?

Reply:
Good idea, I'll look into it!

Reply:
I actually decided to drop the package-manager specific part, including the reference to global turbo, in favor of just demonstrating it. I think we probably want the global turbo piece to live in a more explicit "installing turbo" section